NEWS.md

```diff
@@ -1,4 +1,5 @@
 # namer (development version)
+* unname_all_chunks now accepts argument `ch_n_p` with the prefix of the chunknames to be unnamed
```

Comment:
in this, 
* add your GitHub username and the PR number i.e. `"to be unnamed (@HanOostdijk, #22)"`